src/models/author.js

```diff
@@ -0,0 +1,27 @@
+module.exports = (connection, DataTypes) => {
+  const schema = {
+    author: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        notNull: {
+          args: true,
+          msg: 'We need author name'
+        },
+        notEmpty: {
+          args: true,
+          msg: 'Insert author name'
+        }
+      }
+    }
+  }
+  const RemoveTimeStamp = {
+    defaultScope: {
+      attributes: { exclude: ['createdAt', 'updatedAt'] }
+    },
+    timestamps: false
+  }
+
+  const AuthorModel = connection.define('Author', schema, RemoveTimeStamp)
+  return AuthorModel
```

Comment:
This could be condensed as you are creating a variable that is then never used again. Makes more sense to just return the value you were initially storing in the variable: 
```
  return connection.define('Author', schema, RemoveTimeStamp)
```